.circleci/run.sh

```diff
@@ -113,9 +113,10 @@ coverage()
     if [ -f ~/dlang/install.sh ] ; then
         source "$(CURL_USER_AGENT=\"$CURL_USER_AGENT\" bash ~/dlang/install.sh dmd-$HOST_DMD_VER --activate)"
     fi
+    RDMD="$(type -p rdmd)"
 
     # build dmd, druntime, and phobos
-    make -j$N -C src -f posix.mak MODEL=$MODEL HOST_DMD=$DMD BUILD=$BUILD ENABLE_WARNINGS=1 PIC="$PIC" all
+    "$RDMD" ./src/build.d MODEL=$MODEL HOST_DMD=$DMD BUILD=$BUILD ENABLE_WARNINGS=1 PIC="$PIC" all
```

Comment:
So we slowly start introducing `build.d` into more parts of the CI infrastructure.